cli/dts/lib.deno.unstable.d.ts

```diff
@@ -203,10 +203,67 @@ declare namespace Deno {
    */
   export function consoleSize(
     rid: number,
-  ): {
+  ): ConsoleSize;
+
+  /**
+   * **UNSTABLE**: New API, yet to be vetted.
+   *
+   * Sets the size of the console.
+   *
+   * ```ts
+   * Deno.resizeConsole(Deno.stdout.rid, {
+   *   rows: 20,
+   *   columns: 80,
+   * });
+   * ```
+   */
+  export function resizeConsole(
+    rid: number,
+    size: ConsoleSize,
+  ): void;
+
+  /**
+   * **UNSTABLE**: New API, yet to be vetted.
+   *
+   * Opens a new Pty interface.
+   *
+   * ```ts
+   * const pty = Deno.openPty({
+   *   rows: 20,
+   *   columns: 80,
+   * });
+   *
+   * await Deno.spawn(Deno.execPath(), {
+   *   args: [
+   *     "eval",
+   *     `
+   *       const size = Deno.consoleSize(Deno.stdout.rid);
+   *       console.log("%sx%s", size.columns, size.rows);
+   *     `
+   *   ],
+   *   tty: pty.rid,
```

Comment:
Accept the `Pty` object directly, unwrap the rid when calling the op
```suggestion
   *   tty: pty,
```